hs/src/Reach/Verify/SMT.hs

```diff
@@ -971,25 +976,68 @@ smt_la at_de dla = do
       vv' <- smt_a at_de vv
       return $ smtApply (s ++ "_" ++ vn) [vv']
 
+sub_loop_vars_a :: M.Map DLVar DLArg -> DLArg -> DLArg
```

Comment:
These functions might be `Subst`

Reply:
If I change `SubstEnv` from `Map DLVar DLVar` to `Map DLVar DLArg`, remove the instance`Subst DLVar` and tweak `DeJump` accordingly I can re-use it. Any concern with removing the `subst dlvar` instance?